presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/DockerContainer.java

```diff
@@ -81,6 +91,21 @@ public void copyFileToContainer(Transferable transferable, String containerPath)
         copyFileToContainer(containerPath, () -> super.copyFileToContainer(transferable, containerPath));
     }
 
+    public DockerContainer withExposedLogPaths(String... logPaths)
+    {
+        this.logPaths.addAll(Arrays.asList(logPaths));
+        return this;
+    }
+
+    public void exposeLogsInHostPath(Path hostBasePath)
```

Comment:
`withExposedLogPaths` and `exposeLogsInHostPath` look like configuration methods, but currently need to be called in correct order

let's add some verify to ensure they are always callled in that order.
simplest way would be

- to null out `logPaths` inside `exposeLogsInHostPath`
- to check `logPaths != null` yet in `withExposedLogPaths`

(i don't see a need to support the methods being called in varying order, so let's not spend time on that)